planets.stats.user.js

```diff
@@ -1,22 +1,21 @@
 // ==UserScript==
 // @name         Dark Galaxy - Planets stats
 // @namespace    https://darkgalaxy.com/
-// @version      0.8
+// @version      0.9
 // @description  All your planet are belong to us
 // @author       Biggy
 // @homepage     https://github.com/fl0v/dg
 // @supportURL   https://github.com/fl0v/dg/issues
 // @downloadURL  https://github.com/fl0v/dg/raw/master/planets.stats.user.js
-// @match        https://beta.darkgalaxy.com/planets/
-// @match        https://andromeda.darkgalaxy.com/planets/
+// @match        https://*.darkgalaxy.com/planets/
 // @grant        none
 // ==/UserScript==
 
 (function () {
 
     const resPattern = /([\d,]+)\s+\(([\+\d,]+)\)\s+([\d%]+)/; // will split resource data ex: '52,126 (+3,465) 70%'
-    const popPattern = /([\d,]+)\s+\(([\+\d,]+)\soccupied\)/; // will split population  data ex: '52,126 (5,000 occupied)'
-    const othPattern = /([\d,]+)/; // simple value for other resources
+    const popPattern = /([\d,]+)\s+\/\s+[\d,]+\s+\(([\+\d,]+)\s+available\)/; // will split population  data ex: '52,126 (47,126 available)'
+    const othPattern = /([\d,]+)/; // simple value for other resources (disabled because it does not work)
```

Comment:
i think this was suppose to be for soldiers, ground and orbit which i see you removed.
Did the template change ? can it be fixed?
Or maybe its broken because some edge case?